### ISSUE TITLE
Homme tropical cyclone test has unexplained extremum in ps

### ISSUE THREAD

**Issue by oksanaguba** [E3SM-Project/E3SM#2554]
The test was run with *nl below. It produced frames each 4 hours, and there is another extremum next to the cyclone around day 8. It then gets absorbed, so, if frames are done less frequently, it would not be detected. The pic is attached. First i thought it is due to recent merges, but commit 3758201 from May 2018 has it.

**Comment by oksanaguba** [comment]
Namelist:

&ctl_nl
  nthreads          = 1
  partmethod        = 4                         ! mesh parition method: 4 = space filling curve
  topology          = "cube"                    ! mesh type: cubed sphere
  test_case         = "dcmip2016_test2"         ! test identifier
  ne                = 60                         ! number of elements per cube face
  qsize             = 4                         ! num tracer fields
  ndays             = 10
  statefreq         = 864                       ! number of steps between screen dumps
!  restartfreq       = 8                        ! don't write restart files if < 0
  runtype           = 0                         ! 0 => new run
  tstep             = 50                        ! largest timestep in seconds
  integration       = 'explicit'                ! explicit time integration
  tstep_type        = 5
  rsplit            = 0
  qsplit            = 1
  nu                = 1.1e14                    ! default= 1e15*(ne30/ne60)**3.2 = 1.1e14
  nu_s              = 1.1e14
  nu_p              = 0 !1.1e14
  nu_top            = 0                         ! 2.5e5/(120)^(1)
  limiter_option    = 8
  hypervis_order    = 2                         ! 2 = hyperviscosity
  hypervis_subcycle = 1                         ! 1 = no hyperviz subcycling
  moisture          = 'wet'
  dcmip16_prec_type = 0                         ! 0=kessler,     1= reed-jablonowski
  dcmip16_pbl_type  = 0                         ! 0=basic pbl,   1= bryan pbl
/
&vert_nl
  vform         = "ccm"
  vfile_mid     = "./camm-30.ascii"
  vfile_int     = "./cami-30.ascii"
/
&analysis_nl
  output_dir        = "./movies/"               ! destination dir for netcdf file
  output_timeunits  = 2,                        ! 0=timesteps, 1=days, 2=hours, 3=seconds
  output_frequency  = 4                         ! every 3 hours
  output_varnames1  ='T','ps','u','v','omega','Q','zeta'   ! variables to write to file
  interp_type       = 0                         ! 0=native grid, 1=bilinear
  output_type       ='netcdf'                   ! netcdf or pnetcdf
  num_io_procs      = 16
  output_prefix     ='firstrun-days8-interp-'
!  interp_nlon       = 360
!  interp_nlat       = 181
/
&prof_inparm
  profile_outpe_num   = 100
  profile_single_file   = .true.
/

**Comment by oksanaguba** [comment]
![image](https://user-images.githubusercontent.com/8389408/45924139-59d09500-beb5-11e8-9d16-d80752c1429f.png)

**Comment by oksanaguba** [comment]
Update: First I considered that the second disturbance (artifact) should not be there, because of some movie file from a while ago, where the artifact is not present at day 8, unlike in current master. I found commit that was used to create that old movie and reran it to produce more frames. Shortly, there was a bug in the branch, and it seems that there is nothing wrong with master. Longer explanation is below.

I have three different commits, master 8a9fce30 before vsplit2 branch, current master 67a0f985 (after branch went in) and a branch commit d5394eba from between. Since two master commits for the nl above are bfb, it means that the branch had a buggy code somewhere. The difference between the branch commit and current master is that the artifact is in the branch run, too, but it is presented earlier than at day 8 (8 hours earlier or so). 

Closing this issue. While it is tricky, one could go to dcmip2016 page and try to see whether the artifact is there for other dycores (i am semi-positive it is there for ICON). Thanks to AMB for helping me with this.
